blocks/test/fixtures/core-embed.serialized.html

```diff
@@ -0,0 +1,6 @@
+<!-- wp:core/embed url="https://example.com/" -->
+<figure>https://example.com/
```

Comment:
So this is something I didn't know about JSX... whitespace and newlines are not preserved. I'll have to replace the JSX with normal strings to get the desired output here, and I may as well do it as part of this PR...